ldi-orchestrator/ldio-connectors/ldio-http-in2/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldio/LdioHttpInController.java

```diff
@@ -0,0 +1,46 @@
+package be.vlaanderen.informatievlaanderen.ldes.ldio;
+
+import be.vlaanderen.informatievlaanderen.ldes.ldio.event.HttpInPipelineCreatedEvent;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.context.event.EventListener;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static java.lang.String.valueOf;
+import static java.util.Optional.ofNullable;
+
+@RestController
```

Comment:
As I mentioned yesterday, maybe we want @RequestMapping prefixes here for the different input components